app/controllers/application_controller/policy_support.rb

```diff
@@ -95,6 +95,8 @@ def policy_sim(records = [])
         @refresh_partial = "layouts/policy_sim"
         replace_right_cell
       end
+    else
```

Comment:
Adding a new branch, extra complexity...

Reply:
You are right, so what is your suggestion to avoid this?